Functions/Quantification/xASL_quant_Hct2BloodT1.m

```diff
@@ -109,4 +115,22 @@
 end
 
 
+end
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+function xASL_quant_Hct2BloodT1_printMissingHctWarning(x)
+%xASL_quant_Hct2BloodT1_printMissingHctWarning We may print this message in
+%different cases, which is why this subfunction comes in handy
+
+
+fprintf('\n\n\n');
+fprintf('%s\n', 'WARNING: POSSIBLE CBF BIAS');
+fprintf('\n');
```

Comment:
I would print the subject name?

Reply:
No that's going to be too much hassle. It is printed in the same run, and `xASL_quant_Hct2Blood` is a function without `x`. Shouldnt' be too difficult for the user to find the subject name because it is printed all over right (folder and file names).